projects/aca-testing-shared/src/utilities/repo-client/apis/favorites/favorites-api.ts

```diff
@@ -130,7 +140,7 @@ export class FavoritesApi extends RepoApi {
       };
       return await Utils.retryCall(favorite);
     } catch (error) {
-      // this.handleError(`${this.constructor.name} ${this.isFavoriteWithRetry.name}`, error);
+      // this.handleError(`FavoritesApi isFavoriteWithRetry : catch : `, error);
```

Comment:
should this be commented?

Reply:
Yes. During a good and clean run, this fails if the node is already a favorite. For the purpose of the test, all I care is for the items to be favorite, so these logs would only pollute

Reply:
it would be better to use the Logger with a LOG_LEVEL debug in this case it would be visible only based on the config. Btw if we don’t need it just remove it